tests/phpunit/Compat/WC_Stripe_Subscription_Initial_Test.php

```
@@ -0,0 +1,141 @@
+		// Clean up.
+		remove_filter( 'pre_http_request', [ $this, 'pre_http_request_response_success' ] );
+		WC_Stripe_Helper::delete_main_stripe_settings();
+
+		// Assert: nothing was returned.
+		$this->assertEquals( 'success', $result['result'] );
+		$this->assertArrayHasKey( 'redirect', $result );
+
+		$order = wc_get_order( $order_id );
+
+		$this->assertEquals( 'pi_123abc', $order_helper->get_stripe_intent_id( $order ) );
+	}
```

PHPMD (2.15.0)

[warning] The method test_initial_intent_parameters() has 109 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)